.github/release-please-config.json

```diff
@@ -13,5 +13,13 @@
       ]
     }
   },
+  "changelog-sections": [
+    { "type": "feat", "section": "Features" },
+    { "type": "feature", "section": "Features" },
+    { "type": "fix", "section": "Bug Fixes" },
+    { "type": "perf", "section": "Performance Improvements" },
```

Comment:
Not sure if we need this one but I figured I'd let someone else decide.

Reply:
LGTM